setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="celobox",
+    version="0.1a",
+    packages=find_packages(),
+    install_requires=[
+        'selenium',
+        'requests',
+        'enum34',
+        'pyyaml',
+    ],
+    entry_points={
+        'console_scripts': [
+            'phantompasswd = celobox.phantompasswd:main'
```

Comment:
`phantompasswd` is fine for now, but when this replaces the `passwd.py` runner we'll want to figure out a better CLI script name, since [`passwd`](http://man7.org/linux/man-pages/man1/passwd.1.html) is already a thing on *nix that we'd be best to not shadow.